modules/vault-cluster/variables.tf

```diff
@@ -219,3 +219,14 @@ variable "enabled_metrics" {
   default     = []
 }
 
+variable "enable_dynamo_backend" {
+  description = "Whether to use a DynamoDB storage backend instead of Consul"
+  type        = bool
+  default     = false
+}
+
+variable "dynamo_backend_policy" {
```

Comment:
Why take in the IAM policy as an arg rather than creating as you had before? I mentioned limiting the policy to the minimum set of IAM actions rather than `*`, but that minimum set is well-known for Vault, so it seems worth creating the policy in this module (conditionally) rather than forcing each user to figure it out themselves?

Reply:
Where global tables are used instances would need access to write to tables in their respective regions. 

An alternative is to set the resource on the policy to `arn:aws:dynamodb:*:111111111:table/${var.dynamo_table_name}`. This works since all table names must be the same

Reply:
Ah, I gotcha. Perhaps the module takes in `dynamo_table_name` and `dynamo_table_region` as params and constructs the ARN above... And the variable description could tell the customer to set `dynamo_table_region` to `*` for global tables?